app/script/util/LocalizerUtil.js

```diff
@@ -23,13 +23,13 @@ window.z = window.z || {};
 window.z.util = z.util || {};
 
 z.util.LocalizerUtil = {
-  joinNames: (userEntities, declension = z.string.Declension.ACCUSATIVE) => {
+  joinNames: (userEntities, declension = z.string.Declension.ACCUSATIVE, skipAnd = false) => {
     const firstNames = userEntities
       .map(userEntity => z.util.getFirstName(userEntity, declension))
       .sort((userNameA, userNameB) => z.util.StringUtil.sortByPriority(userNameA, userNameB));
 
     const numberOfNames = firstNames.length;
-    if (numberOfNames >= 2) {
+    if (!skipAnd && numberOfNames >= 2) {
       const [secondLastName, lastName] = firstNames.splice(firstNames.length - 2, 2);
```

Comment:
Here we sort the names. You/du is the first name of self and will be sorted accordingly. Thus self will not always be last..

Reply:
You're right, we should be able to skip this